src/main/java/io/openliberty/tools/intellij/actions/RunLibertyDevTask.java

```diff
@@ -57,8 +58,9 @@ public void update(@NotNull AnActionEvent e) {
             JComponent libertyWindow = content.getComponent();
             Component[] components = libertyWindow.getComponents();
             for (Component comp : components) {
-                if (comp.getName() != null && comp.getName().equals(Constants.LIBERTY_TREE)) {
-                    Tree libertyTree = (Tree) comp;
+                if (comp instanceof JBScrollPane && comp.getName() != null && comp.getName().equals(Constants.LIBERTY_SCROLL_PANE)) {
+                    JBScrollPane scrollPane = (JBScrollPane) comp;
+                    Tree libertyTree = (Tree) scrollPane.getViewport().getView();
```

Comment:
As Aparna recommended in her review, it would be best to check the type of `scrollPane.getViewport().getView()` before typecasting as a `Tree`.